CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+# dbt_pinterest_source v0.10.0
+[PR #28](https://github.com/fivetran/dbt_pinterest_source/pull/28) includes the following updates:
+
+## Breaking changes
```

Comment:
Additionally it looks like you changed the schema and database variables from `pinterest_schema` to `pinterest_ads_schema` and `pinterest_database` to `pinterest_ads_database` respectively. 

This is a major breaking change. We should highlight this as the first bullet in the CHANGELOG.

Reply:
I was thinking to standardize all the names, but since it is a big change, I see what you mean. I have put the schema and database variables back.